plugins/dev-tools/src/addGUIControls.js

```diff
@@ -252,11 +241,40 @@ function populateRendererOption(folder, options, onChange) {
   // rendererMap_, whereas newer versions that use the global registry get their
   // list of renderers from somewhere else.
   const renderers = Blockly.blockRendering.rendererMap_ ||
-      Blockly.registry.typeMap_['renderer'];
+      (Blockly.registry && Blockly.registry.typeMap_['renderer']);
   folder.add(options, 'renderer', Object.keys(renderers))
       .onChange((value) => onChange('renderer', value));
 }
 
+/**
+ * Populate the theme option.
+ * @param {dat.GUI} folder The dat.GUI folder.
+ * @param {Blockly.Options} options Blockly options.
+ * @param {Blockly.Options} defaultOptions Default Blockly options.
+ * @param {function(string, string):void} onChange On Change method.
+ */
+function populateThemeOption(folder, options, defaultOptions, onChange) {
+  let themes;
+  if (Blockly.registry && Blockly.registry.typeMap_['theme']) {
+    // Using a version of Blockly that registers themes.
+    themes = Blockly.registry.typeMap_['theme'];
```

Comment:
Should we make a public getter for this?

Reply:
Aside from dev purposes, which is what we're doing here. What would be the use case for a public getter?

Reply:
Someone could want to display all the themes to the user so they can choose in a drop down which one they want.

Reply:
Sure, could be useful. Although in cases like that I would resort to maintaining my own list of themes I want to display to my users. eg, just because the Dark theme is "available" (ie; in the registry), doesn't mean the developer wants to show that to their users.

Filed issue https://github.com/google/blockly/issues/3935